pkg/virtctl/expose/expose.go

```diff
@@ -261,3 +292,24 @@ func podNetworkPorts(vmiSpec *v12.VirtualMachineInstanceSpec) []v1.ServicePort {
 	}
 	return nil
 }
+
+func serverVersion(virtClient kubecli.KubevirtClient) (major int, minor int, err error) {
+	serverVersion, err := virtClient.DiscoveryClient().ServerVersion()
+	if err != nil {
+		return 0, 0, err
+	}
+	// Make a Regex to say we only want numbers
+	reg, err := regexp.Compile("[^0-9]+")
+	if err != nil {
+		return 0, 0, err
+	}
+	major, err = strconv.Atoi(reg.ReplaceAllString(serverVersion.Major, ""))
+	if err != nil {
+		return 0, 0, err
+	}
+	minor, err = strconv.Atoi(reg.ReplaceAllString(serverVersion.Minor, ""))
+	if err != nil {
+		return 0, 0, err
+	}
+	return
```

Comment:
we're attempting to parse the semver here, right?

Take a look at the usage of the semver package in ```pkg/virt-operator/install-strategy/create.go```.  There's already a way to parse a semver from a string and get the major/minor from it.

Reply:
Thanks for review it, I will do like the installer.

Reply:
@davidvossel thinking about it we don't try to parse it, the major/minor are already splited at `Major`, `Minor` fields but sometime they got special chars like Major=1, Minor=17+, so using the approach from install-strategy does not give much value here, what do you think ?